tests/dragonfly/utility.py

```diff
@@ -214,17 +227,33 @@ def rand_str(k=3, s=""):
 
         if t == ValueType.STRING:
             # Random string for MSET
-            return (rand_str(self.val_size),)
+            return (rand_str(self.huge_val_size if generate_huge_val else self.val_size),)
         elif t == ValueType.LIST:
             # Random sequence k-letter elements for LPUSH
-            return tuple(rand_str() for _ in range(self.val_size // 4))
+            list_size = self.val_size // 4
+            huge_val_element = random.randint(0, list_size - 1) if generate_huge_val else -1
```

Comment:
can we do 
element_size = self.huge_val_size / list_size if generate_huge_val else self.val_size / list_size
and call rand_str(element_size) for all the cases below ?

Reply:
Different types will have different element size produced (i.e. list will have 4, hset 5, and so on). With this PR we are selecting *only one* element to have huge string (not all). Purposed change will produce all elements to have *huge_val // list_size*  or chars (if this is expected i can provide PR that change this).

Reply:
I am not sure why the default was 3 before. 
IMO it makes sense that the sum of str length of elements in the container will be val_size where val size is either self.val_size or self.huge_val_size

Reply:
going to update to this behaviour